src/stories/Pill.stories.tsx

```diff
@@ -0,0 +1,103 @@
+import { storiesOf } from '@storybook/react';
+import * as React from 'react';
+import { Pill } from '../components';
+import UnbraceThemeProvider from '../components/UnbraceThemeProvider';
+
+const propsStoryConfig = {
+  props: {
+    propTablesExclude: [UnbraceThemeProvider],
+  },
+};
+
+storiesOf('Pill', module).add(
+  'colors',
+  () => {
+    return (
+      <UnbraceThemeProvider>
+        <React.Fragment>
+          <Pill size="m" color="neutral" style={{ marginRight: 8 }}>
```

Comment:
can marginRight not better be added to the styles of the component itself 18 times the same styling seems a bit too much, or do we prefer it this way since it's just used for storybook component?

Reply:
I don't think the margin should be present on the component itself, like you said, it is just for the storybook component. This way you have full control of the spacing around the component when you're using it.